examples/tensorflow_eager_simple.py

```diff
@@ -20,15 +20,18 @@
 
 import tensorflow as tf
 from tensorflow.keras.datasets import mnist
+import pkg_resources
```

Comment:
Please reorder the imports in alphabetical order.

https://circleci.com/gh/optuna/optuna/26154
```
flake8 .
./examples/tensorflow_eager_simple.py:23:1: H306: imports not in alphabetical order (tensorflow.keras.datasets.mnist, pkg_resources)
1     H306: imports not in alphabetical order (tensorflow.keras.datasets.mnist, pkg_resources)
```

Reply:
Thanks! I fixed it.